controllers/ControllerUser.class.php

```diff
@@ -49,8 +49,29 @@ public function logout() {
 
     public function settings() {
         $error = null;
+        if (!isset($_SESSION['userId'])) {
+            throw new Exception('You need to be connected to access the settings !');
```

Comment:
```suggestion
            throw new Exception('You need to be connected to access the settings !');
```
The text could be improve 'You must be connected'.